### PR TITLE
created notes listing page ui

### LINKED ISSUE #3: 2 - Create note listing page UI

### PR DESCRIPTION
**Pull request by adityamahajan-io**
closes #3

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/Note.jsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        {note.description}
+      </Typography>
+    </div>
+    <hr />
+    <div className=" mt-3 flex justify-between align-middle">
+      <Tag className="bg-gray-100" label="Getting Started" size="small" />
+      <div className="flex items-center space-x-2">
+        <Clock size={18} />
+        <Tooltip
+          content={dayjs(note.created_at).format("dddd, hh:mmA")}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Note.jsx:35]: You can move this to a well name util function and move it to a utils.js within the closest scope. We should try our level best to keep the presentation logic as clean as possible.

File: app/javascript/src/components/Dashboard/Notes/Note.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          position="bottom"
+        >
+          <Typography style="body3">Created 2 Hours Ago</Typography>
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/Note.jsx:38]: Create a util function for this. I believe this value should be presented based on the backend column value of `created_at`.

File: app/javascript/src/components/Dashboard/Notes/index.jsx [hunk 3]
```diff
@@ -41,62 +42,58 @@ const Notes = () => {
...
-          image={EmptyNotesListImage}
-          primaryAction={() => setShowNewNotePane(true)}
-          primaryActionLabel="Add New Note"
-          subtitle="Add your notes to send customized emails to them."
-          title="Looks like you don't have any notes!"
+    <>
+      <MenuBar showMenu={showMenuBar} />
+      <Container>
+        <Header
+          menuBarToggle={() => setShowMenuBar(!showMenuBar)}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/index.jsx:49]: https://academy.bigbinary.com/learn-react/pages/know-when-to-use-previous-state-for-state-update

**Review by yedhink — COMMENTED** [review]
@adityamahajan-io _a Few comments added.

**Comment by adityamahajan-io** [comment]
@yedhink I have fixed the reviews mentioned above in PR #8. Please have a look.

**Pull request merged**